sdk/js/src/types.ts

```diff
@@ -178,6 +189,11 @@ export interface DVCClient {
      */
     allVariables(): DVCVariableSet
 
+    /**
+     * Retrieve configuration settings related to Opt-in
+     */
+    getOptInSettings(): OptInSettings
```

Comment:
we should be able to change this to:
```suggestion
    getOptInSettings(): Project["settings"]["optIn"]
```
instead of creating a new type definition